py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,40 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
+    format_device = '{name}'
+
+    class Meta:
+        def deprecate_function(config):
+            out = {}
+            if 'format_prefix' in config:
+                out['format'] = u'{}{{format_device}}'.format(config['format_prefix'])
+            return out
```

Comment:
this is a good start.  I think the format should still be None as the default then we can check if the user has set it etc.

And we need some more checks etc

If you want I'll knock up a patch.  Won't be today as I'm off out soon